plugins/toolkit/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```
@@ -591,12 +591,16 @@ codemodernizer.chat.message.transform_cancelled_by_user=I cancelled your transfo
 codemodernizer.chat.message.transform_in_progress=If I run into any issues, I might pause the transformation to get input from you on how to proceed.
 codemodernizer.chat.message.transform_stopped_by_user=I stopped your transformation. If you want to start another transformation, choose **Start a new transformation**.
 codemodernizer.chat.message.transform_stopping=I'm stopping your transformation...
+codemodernizer.chat.message.upload_failed_connection_refused=Sorry, I couldn't upload your project to begin the transformation. Please check your network connectivity or firewall configuration, and then try again.
+codemodernizer.chat.message.upload_failed_http_error=I'm sorry, I couldn't upload your project to start the transformation. You can investigate the issue with the HTTP Status Code: {0}
```

if the apostrophe is missing:
```suggestion
codemodernizer.chat.message.upload_failed_http_error=I''m sorry, I couldn't upload your project to start the transformation. You can investigate the issue with the HTTP Status Code: {0}
```

---

I will remove the apostrophe from error messages